management/README.md

```diff
@@ -1,7 +1,8 @@
-# Azure Service Management packages for Go (DEPRECATED)
+# Azure Service Management packages for Go
 
 The `github.com/Azure/azure-sdk-for-go/management` packages are used to perform operations using the Azure Service Management (ASM), aka classic deployment model. Read more about [Azure Resource Manager vs. classic deployment](https://azure.microsoft.com/documentation/articles/resource-manager-deployment-model/). Packages for Azure Resource Manager are in the [arm](../arm) folder.
 Note that this package requires Go 1.7+ to build.
+This package is in mainteinance mode and won't receive any future updates. It is recommended to [migrate to Azure Resource Manager](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-manager-deployment-model) deployment model.
```

Comment:
A fair point, we could change it to say "... won't receive any future updates except for security fixes."